larky/src/main/java/com/verygood/security/larky/objects/LarkyPyObject.java

```diff
@@ -9,6 +9,9 @@
 import com.verygood.security.larky.objects.type.LarkyType;
```

Comment:
There should be a brief to distinguish between this new Larky objects. Some scheme